src/components/AvatarCropModal/ImageCropView.tsx

```diff
@@ -1,60 +1,66 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
 import {PanGestureHandler} from 'react-native-gesture-handler';
+import type {GestureEvent, PanGestureHandlerEventPayload} from 'react-native-gesture-handler';
 import Animated, {interpolate, useAnimatedStyle} from 'react-native-reanimated';
 import Icon from '@components/Icon';
 import * as Expensicons from '@components/Icon/Expensicons';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import ControlSelection from '@libs/ControlSelection';
-import gestureHandlerPropTypes from './gestureHandlerPropTypes';
+import type {SelectionElement} from '@libs/ControlSelection/types';
+import type IconAsset from '@src/types/utils/IconAsset';
 
-const propTypes = {
+type ImageCropViewProps = {
     /** Link to image for cropping   */
-    imageUri: PropTypes.string,
+    imageUri: string;
```

Comment:
```suggestion
    imageUri?: string;
```

According to propTypes.